Sources/Orbit/Components/HorizontalScroll.swift

```diff
@@ -1,19 +1,11 @@
 import SwiftUI
+import Combine
 
 public enum HorizontalScrollItemWidth {
-    /// Each item will use its own intrinsic width.
-    case intrinsic
-    /// Width ratio compared to container width.
-    case ratio(CGFloat = 0.8)
+    /// Width ratio calculated from the available container width.
+    case ratio(CGFloat = 0.48, maxWidth: CGFloat? = Layout.readableMaxWidth - 270)
```

Comment:
The maxWidth was already there, just have been moved here. It is there as a cap for regular horizontal presentation (so that the call site does not have to switch over it and keep the same ratio).

The new 0.48 is a ratio that fits 2 items and makes a 3rd item edge visible. This works as a better defaults for both compact and regular widths.

For example, a ratio of 0.5 or 1.0 would fit the items exactly, but the edges of next scrollable items would not be visible.